tests/view/test_extents.cpp

```diff
@@ -0,0 +1,71 @@
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+#include "Kokkos_Core.hpp"
+
+#include "kokkos-utils/array/compare.hpp"
+#include "kokkos-utils/view/extents.hpp"
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+/**
+ * @file
+ *
+ * @addtogroup unittests
+ *
+ * **Get extents of a @c Kokkos::View**
+ *
+ * This group of tests check the behavior of our helpers related to @c Kokkos::View that can be
+ * found in @ref extents.hpp.
+ */
+
+namespace Kokkos::utils::tests::view
+{
+
+constexpr size_t dim_1 = 5, dim_2 = 3, dim_3 = 6, dim_4 = 9;
+
+#define CHECK_EXTENTS(__view__, __expted__) ASSERT_TRUE(utils::view::extents(__view__) == __expted__)
```

Comment:
Could we avoid the new macro and use `ASSERT_EQ` directly?

- https://google.github.io/googletest/reference/assertions.html#EXPECT_EQ

E.g., it would be  `ASSERT_EQ(view_1_static, Kokkos::Array{dim_1});` and so on. Seems just a short :)

Reply:
Let's try.

Reply:
Looks like it worked!